ide-migration/server/migration/api/migration-service.js

```diff
@@ -270,7 +271,12 @@ class MigrationService {
         const project = workspace.getProject(projectName)
         const projectFile = project.createFile(relativePath);
         const resource = repositoryManager.getResource(repositoryPath);
-        projectFile.setContent(resource.getContent());
+
+        if (relativePath.endsWith('hdbcalculationview') || relativePath.endsWith('calculationview') || repositoryPath.endsWith('hdbcalculationview') || repositoryPath.endsWith('calculationview')) {
+            projectFile.setContent(calculationViewModifier.removeTypeArtifact(resource.getContent()));
+        } else {
+            projectFile.setContent(resource.getContent());
+        }
```

Comment:
FYI: @vmutafov, @ivanvolkoff

Ideally this logic block should be extracted into separate process tasks, as it's expected to have also several others transformations happening as part of the file content migration/modification.

Reply:
We are currently working on extracting the logic for transformation/generation files to a different place.

Reply:
@ivanvolkoff Great! Should we merge the PR, or there are upcoming changes that we should wait for?

Reply:
I merged this one, I’ll create a PR for the XSK right after lunch

Reply:
Thanks @ivanvolkoff and @vmutafov!